src/runtime_src/core/tools/common/ReportAie.cpp

```diff
@@ -102,8 +103,8 @@ ReportAie::writeReport(const xrt_core::device* /*_pDevice*/,
       for (auto& node : graph.get_child("tile")) {
         const boost::property_tree::ptree& tile = node.second;
 
-	if (tile.get<std::string>("memory_column", "") == "")
-	  continue;
+	      if (tile.get<std::string>("memory_column", "") == "")
```

Comment:
Why the extra spaces?

Reply:
It is because of previous author code push, I just moved the function. I will fix it.